integration-tests/elytron-security-oauth2/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-elytron-security-oauth2</artifactId>
+    <name>Quarkus - Integration Tests - Elytron Security OAuth2</name>
+
+    <properties>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-servlet</artifactId>
+                <version>9.4.17.v20190418</version><!-- or 9.4.15.v20190215 -->
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-elytron-security-oauth2</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Your extension deployment and runtime modules should be in the dependencyManagement in the bom/pom/xml I believe

Reply:
done.
I dind't add it in the extension.json file, I don't know if it needs to be done yet or not.

Reply:
@loicmathieu thanks, my understanding it has to be done from the start so please update the extensions.json

Reply:
done